dds_web/api/files.py

```diff
@@ -553,10 +554,10 @@ def put(self, _, project):
             file.time_latest_download = dds_web.utils.timestamp()
         except sqlalchemy.exc.SQLAlchemyError as err:
             db.session.rollback()
-            app.logger.exception(str(err))
+            flask.current_app.logger.exception(str(err))
             return flask.make_response(str(err), 500)
```

Comment:
This is something we need to look into too. What info to display and not display to the end user.

Reply:
Yes, I don't think the error has to be very informative unless it can be predicted, basically giving a standard "500 Internal Server Error" in other cases.